nncf/onnx/hardware/fused_patterns.py

```diff
@@ -11,63 +11,518 @@
  limitations under the License.
```

Comment:
@kshpv, please, review this file in detail. Also, please, explain if we really need these massive amounts of pattern combinations for ONNX.

Reply:
cc @andrey-churkin, @alexsu52, @vshampor